index.html

```diff
@@ -0,0 +1,96 @@
+            <path d="M 23.857422 8.5 C 17.504717 8.5 11.602344 8.9526234 8.234375 9.65625 A 1.50015 1.50015 0 0 0 8.2128906 9.6621094 C 5.6754768 10.230693 3.2861897 12.048234 2.7832031 14.894531 A 1.50015 1.50015 0 0 0 2.78125 14.90625 C 2.394836 17.200265 2 20.190694 2 24.5 C 2 28.801151 2.3961903 31.712324 2.8847656 34.126953 C 3.4000756 36.889296 5.7342165 38.761817 8.3105469 39.337891 A 1.50015 1.50015 0 0 0 8.3476562 39.345703 C 9.4934962 39.571703 10.867317 39.770021 12.427734 39.9375 A 1.50015 1.50015 0 1 0 12.748047 36.955078 C 11.263916 36.795799 9.9775527 36.608469 8.9570312 36.408203 C 7.3549636 36.046839 6.0823148 34.927476 5.8320312 33.572266 A 1.50015 1.50015 0 0 0 5.8261719 33.546875 C 5.3660305 31.276194 5 28.628694 5 24.5 C 5 20.378688 5.3654221 17.62199 5.7363281 15.417969 C 5.9947549 13.955585 7.2164425 12.963194 8.859375 12.591797 C 11.774266 11.984227 17.659955 11.5 23.857422 11.5 C 25.529327 11.5 27.167838 11.533447 28.734375 11.591797 A 1.50015 1.50015 0 1 0 28.845703 8.59375 C 27.24424 8.5340995 25.569516 8.5 23.857422 8.5 z M 36.109375 9.0683594 A 1.50015 1.50015 0 0 0 35.970703 12.064453 C 37.116939 12.175072 38.116066 12.292711 38.960938 12.410156 C 40.556607 12.774779 41.824643 13.888857 42.074219 15.240234 A 1.50015 1.50015 0 0 0 42.078125 15.265625 C 42.543492 17.56209 42.996187 20.292628 42.998047 24.384766 C 42.904597 28.49001 42.450899 31.244675 42.070312 33.585938 C 41.810413 35.044446 40.592 36.034419 38.953125 36.40625 C 35.805209 37.023818 30.142051 37.5 23.951172 37.5 C 22.088453 37.5 20.276688 37.4588 18.566406 37.382812 A 1.50015 1.50015 0 1 0 18.433594 40.378906 C 20.189312 40.456916 22.043891 40.5 23.951172 40.5 C 30.303877 40.5 36.042686 40.04028 39.558594 39.347656 A 1.50015 1.50015 0 0 0 39.595703 39.337891 C 42.133117 38.769306 44.522404 36.951766 45.025391 34.105469 A 1.50015 1.50015 0 0 0 45.029297 34.083984 C 45.409789 31.743169 45.902812 28.755621 46 24.439453 A 1.50015 1.50015 0 0 0 46 24.40625 C 46 20.087697 45.50571 17.078675 45.023438 14.695312 C 44.512192 11.927074 42.175378 10.049478 39.595703 9.4726562 A 1.50015 1.50015 0 0 0 39.476562 9.4511719 C 38.561473 9.3223585 37.478123 9.1958921 36.257812 9.078125 A 1.50015 1.50015 0 0 0 36.109375 9.0683594 z M 20.460938 16.021484 C 18.668395 16.079653 17 17.523394 17 19.484375 A 1.50015 1.50015 0 1 0 20 19.484375 C 20 19.055016 20.359545 18.843206 20.761719 19.070312 L 28.751953 23.583984 C 29.157705 23.813217 29.157705 24.18483 28.751953 24.414062 L 20.761719 28.929688 C 20.359545 29.156793 20 28.943031 20 28.513672 L 20 24 A 1.50015 1.50015 0 1 0 17 24 L 17 28.513672 C 17 31.128313 19.966455 32.823909 22.238281 31.541016 L 30.228516 27.027344 C 32.516764 25.734577 32.516764 22.265423 30.228516 20.972656 L 22.238281 16.458984 A 1.50015 1.50015 0 0 0 22.238281 16.457031 C 21.670325 16.136308 21.058451 16.002095 20.460938 16.021484 z"></path>
+            </svg></a>
+    </div>
+    <h4>&#169 2024 Mondelez</h4>

```

Comment:
Caracterele speciale trebuie să se termine cu `;`.

```suggestion
    <h4>&#169; 2024 Mondelez</h4>
```

Dar există caracter special mai sugestiv.
```suggestion
    <h4>&copy; 2024 Mondelez</h4>
```